src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -23,6 +23,7 @@ import { Subject } from 'rxjs/Subject';
 import 'rxjs/add/operator/takeUntil';
 import { AppConfig } from 'app/app.config';
 import { SuppressKeyboardEventParams } from 'ag-grid/src/ts/entities/colDef';
+import value from '*.json';
```

Comment:
remove unused import and also look for other places.

Reply:
removed